http/http-advanced/src/test/java/io/quarkus/ts/openshift/http/clients/HealthClientServiceBuilder.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.ts.openshift.http.clients;
+
+import java.io.IOException;
+import java.net.URI;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+
+public class HealthClientServiceBuilder  extends CommonClientService {
+
+    public static HealthClientServiceBuilder build(String uri) {
+        return new HealthClientServiceBuilder(uri);
+    }
+
+    private HealthClientServiceBuilder(String uri) {this.uri = uri;}
+
+    public HealthClientService build() throws CertificateException, NoSuchAlgorithmException, KeyStoreException, IOException {
```

Comment:
This code is duplicated in several places. We could move it into CommonClientService.

Reply:
it's ok is not exactly the same.

Reply:
I don't see many differences but the last statement: `return builder.build(HealthClientService.class);`

We could move the other code into the parent class and this method to get a Class<T> parameter:

```
public T createService(Class<T> clazz) {
        RestClientBuilder builder = RestClientBuilder.newBuilder().baseUri(URI.create(this.uri));
        builder.hostnameVerifier((s, sslSession) -> this.hostVerified);
        if (!StringUtils.isEmpty(this.keyStorePath)) {
            builder.trustStore(loadKeystore());
        }

        return builder.build(clazz);
```